docs(relocation): Refresh Moving to SaaS for new flow (#9816)
getsentry/sentry-docs: docs/product/accounts/migration/index.mdx

## Relocating to Cloud

If your organization started with self-hosted Sentry, you've invested time and effort in configuring and customizing your Sentry org. Once you decide to move to a SaaS account, it's important to know that you can relocate a lot of that investment into your new account instead of starting from scratch.

To make this relocation as seamless as possible, Sentry has a workflow that allows you to retain most of your account's settings and configurations as you move to SaaS. All you'll need is owner access to your self-hosted install, and a way to access it via the terminal.

### What you'll keep

All of the configurations, team and project structure, and most metadata will be relocated. Specifically, for every organization you [choose to relocate](#2-choose-which-organizations-to-relocate) this includes:

- All of the organization settings
- The user accounts and options for all of the organization's members
- Projects and their settings
- Teams and their settings
- Alert rules
- Crons
- Dashboards
- Enabled data scrubbers
- Repositories
- Recent and saved searches

### What gets left behind

> Any data related to specific issues, events, stack traces, or files like replays and source maps won't be coming along. Additionally, for security reasons, you'll need to reconnect and re-authenticate all of your integrations, and your users will need to re-connect their 2FA and mint new API tokens after relocating.

Reviewer: ```suggestion
Any data related to specific issues, events, stack traces, or files like replays and source maps won't be transferred. Additionally, for security reasons, you'll need to reconnect and re-authenticate all of your integrations, and your users will need to re-connect their 2FA and create new API tokens after relocating.
```

We try to avoid language that may not be as easy to understand for non native English speakers :)

One thing to note is that while we won't be able to transfer over historical event data, new incoming events will populate the projects in your relocated organization as soon as you [update your DSNs](#5-update-your-dsns). All relocated organization members will be automatically notified and re-invited via email once the relocation completes.

### Version support window

> Self-serve relocation is only possible from any of the last two minor self-hosted releases. That is, if the latest self-hosted release is 24.2.3, your self-hosted install must be on at least 24.1.0 to be successfully relocate.

Reviewer: ```suggestion
Self-serve relocation is only possible from the two most up-to-date minor self-hosted releases. That is, if the latest self-hosted release is version 24.2.3, your self-hosted install must be on at least version 24.1.0 to be successfully relocated.
```

> If you'd like to relocate from an earlier release, we ask that you [update](https://develop.sentry.dev/self-hosted/releases/) your self-hosted install to the latest version first. If there is some reason that this is not possible, please [contact support](https://sentry.io/contact/enterprise/) for more guidance.

Reviewer: ```suggestion
If you'd like to relocate from an earlier release, we ask that you [update](https://develop.sentry.dev/self-hosted/releases/) your self-hosted install to the latest version first. If there's some reason that this isn't possible, please [contact support](https://sentry.io/contact/enterprise/) for more guidance.
```

### Data security

> Your data will be encrypted end-to-end during the relocation process. The only file that will pass over the network will be an encrypted tarball that we'll never store in plaintext, and will delete within 30 days of the start of your relocation.

Reviewer: ```suggestion
Your data will be encrypted end-to-end during the relocation process. The only file that will pass over the network will be an encrypted tarball that we'll never store in plaintext. We'll delete it within 30 days of the start of your relocation.
```

## How to Relocate

### 1. Sign up with a promo code

> Before you begin the relocation process, [sign up](https://sentry.io/signup/relocate/) for a new account on Sentry. If you already have a Sentry account, login and navigate to the [relocation landing page](https://sentry.sentry.io/relocation/). In either case, you should see a page like this:

Reviewer: ```suggestion
Before you begin the relocation process, [sign up](https://sentry.io/signup/relocate/) for a new account on Sentry. If you already have a Sentry account, log in and navigate to the [relocation landing page](https://sentry.sentry.io/relocation/). In either case, you should see the page below. **To receive a $100 credit, use the promo code `relocate100`.** 
```

![Relocation getting started page](./img/relocation-landing-page.png)

> **To receive a $100 credit, make sure to use the promo code `relocate100` on this page.**

Reviewer: ```suggestion

```

### 2. Choose which organizations to relocate

> Your self-hosted installation may have had multiple organizations, each identified by a unique slug. You'll need to collect those from your installed instance, and decide which ones you'll want to keep. You can always change these to whatever you like post-relocation on the organization settings page.

Reviewer: ```suggestion
Your self-hosted installation may have multiple organizations, each identified by a unique slug. You'll need to decide which org slugs to keep. You can always change them post-relocation by going to the **Organization Settings** page.
```

![Changing the org slug on the settings page](./img/org-slug.png)

> ### 3. Follow the relocation instructions

Reviewer: ```suggestion
### 3. Follow the in-app relocation instructions
```

> The remaining pages in the relocation instructions will contain specific details on how to export your self-hosted instance's relocatable data. This will require terminal access to the machine running your self-hosted instance. In brief, you'll save a public key that we generate for you on your self-hosted machine, which you'll then use to execute a command that we provide to generate an encrypted tarball containing your data. You'll then upload this tarball, and we'll take it from there!

Reviewer: ```suggestion
The rest of the in-app relocation instructions will walk you through the specific details of how to export your self-hosted instance's relocatable data. This will require terminal access to the machine running your self-hosted instance. Here's a quick overview:

- You'll be asked to save a public key (that we generate on your self-hosted machine)
- You'll then use this key to execute a command which will generate an encrypted tarball with all your data
-Lastly, you'll be prompted to upload this tarball 
```

Most relocations complete in under an hour, though certain large or complex cases can take a bit longer. If you don't get an update from us within 24 hours, please [contact support](https://sentry.io/contact/enterprise/) for further assistance.

### 4. Choose a plan

> Once your relocation succeeds, you can take a look at our [plans](https://sentry.io/pricing/) and select one that suits your needs based on the different features and capabilities. Note that plan pricing varies based on the type of _Contract_ and _Monthly Event Volume_, so make sure you are familiar with your current/expected monthly usage (number of events per month).
> 
> If you're expecting higher volumes or you're interested in our Enterprise capabilities (priority support and SLAs, and single-tenant options, for example), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.

Reviewer: ```suggestion
Once your relocation is complete, you can choose the [pricing plan](https://sentry.io/pricing/) that best suits your needs. Note that plan pricing varies based on your [Event Volume](/product/accounts/quotas/).

If you're expecting higher volumes or you're interested in our Enterprise capabilities (priority support and SLAs, and single-tenant options, for example), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.
```
I think it's better to send them to our quota management docs in case they're not familiar with their expected monthly usage :)

### 5. Update your DSNs

> Though your project data has been fully relocated, the DSNs for all of your projects will have changed, as they are now located at https://sentry.io rather than your prior self-hosted domain name. This means that you'll need to update the [DSN](/product/sentry-basics/dsn-explainer/) you use in each of your SDKs for each relevant project. Once you complete this step, you'll should start to see issues and events (though hopefully not too many!) appearing as before.

Reviewer: ```suggestion
After the move, your project data will be fully relocated, but the DSNs for all of your projects will now be located in https://sentry.io instead of your prior self-hosted domain name. This means that you'll need to update the [DSN](/product/sentry-basics/dsn-explainer/) you use in each of your SDKs for each relevant project. Once you complete this step, you should start seeing issues and events appearing as before.
```
